Cargo.lock

```diff
@@ -113,7 +113,7 @@ version = "0.7.6"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "fcb51a0695d8f838b1ee009b3fbf66bda078cd64590202a864a8f3e8c4315c47"
 dependencies = [
- "getrandom 0.2.7",
+ "getrandom 0.2.8",
```

Comment:
Could you revert those bumping of all deps, please? Better to do that in a separate PR to not mix things. Or do you need some new versions of deps?

Reply:
I didn't change this. I think this goes automatically :thinking:

Reply:
You can revert this file then=)